routes/render.go

```diff
@@ -307,6 +309,13 @@ func render(w http.ResponseWriter, r *http.Request, name string, data interface{
 
 		// TODO State is a token to protect the user from CSRF attacks.
 		args.LogInURL = config.AuthCodeURL("")
+	} else if r.URL.Path == "/golfer/settings" {
```

Comment:
I'd rather this logic moved to the controller, there's no need for it to live in render.go afaict